place_tracker/lib/stub_data.dart

```diff
@@ -96,4 +96,10 @@ class PlaceStubData {
         category: PlaceCategory.wantToGo,
         starRating: 4),
   ];
+
+  static const List<String> reviews = [
```

Comment:
`reviews` isn't that specific a property name since this holds not whole reviews, but just the text for them. It would likely be better as `reviewStrings`, `reviewTexts`, or something similar.